ext/session/session.c

```diff
@@ -789,6 +789,44 @@ static PHP_INI_MH(OnUpdateSidBits) /* {{{ */
 }
 /* }}} */
 
+static PHP_INI_MH(OnUpdateSessionGcProbability) /* {{{ */
+{
+    SESSION_CHECK_ACTIVE_STATE;
+    SESSION_CHECK_OUTPUT_STATE;
+
+    zend_long tmp = zend_ini_parse_quantity_warn(new_value, entry->name);
+
+    if (tmp < 0) {
+        php_error_docref(NULL, E_WARNING, "session.gc_probability must be non-negative");
```

Comment:
Standard wording, as "non-negative" is really confusing, and what "negative" means depends on the language.
```suggestion
        php_error_docref("session.gc_probability", E_WARNING, "session.gc_probability must be greater than or equal to 0");
```

Reply:
Thank you. Added